test/store/layers/layers.reducer.test.js

```diff
@@ -83,59 +80,57 @@ describe('layersReducer', () => {
 	});
 
 	it('sorts an array based on the the \'zIndex\' property', () => {
-		const layerProperties0 = { ...createDefaultLayerProperties(), label: 'label0', zIndex: 2 };
-		const layerProperties1 = { ...createDefaultLayerProperties(), label: 'label1', zIndex: 0 };
-		const layerProperties2 = { ...createDefaultLayerProperties(), label: 'label2', zIndex: 1 };
+		const layerProperties0 = { ...createDefaultLayerProperties(), id: 'label0', zIndex: 2 };
+		const layerProperties1 = { ...createDefaultLayerProperties(), id: 'label1', zIndex: 0 };
+		const layerProperties2 = { ...createDefaultLayerProperties(), id: 'label2', zIndex: 1 };
```

Comment:
`id: 'id0'`  ....to prevent any missleading guesses of readers of this tests in the future (something like: 'Ids are used as labels')